public/index.html

```diff
@@ -116,6 +116,7 @@ <h3 class="fp-logo"><a href="/feed"><i class="material-icons">photo</i> Friendly
           <li class="mdl-menu__item"><i class="material-icons">perm_contact_calendar</i> About - Help - Contact</li>
         </a>
         <li class="fp-sign-out mdl-menu__item fp-signed-in-only"><i class="material-icons">exit_to_app</i> Sign out</li>
+        <li class="fp-delete-account mdl-menu__item fp-signed-in-only"><i class="material-icons">cancel</i> Delete account</li>
```

Comment:
Could you also add a new "Delete account" button in the drawer?

See: https://github.com/firebase/friendlypix-web/blob/e07c615f0dddfd9bea1918926ed97a270de510de/public/index.html#L139-L156

The side drawer shows up in smaller screens (the app has a reactive UI) you can just make your browser window narrower to see it :)